Allure.NUnit.AsyncTests/AsyncOneTimeSetup.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Allure.Net.Commons;
+using NUnit.Framework;
+
+namespace Allure.NUnit.AsyncTests
+{
+    [SetUpFixture]
+    public class AsyncOneTimeSetup
+    {
+        [OneTimeSetUp]
+        public void AsyncSetup()
```

Comment:
This example can be merged with AsyncLifeCycleTests

Reply:
Moved this initialization to `[AllureNUnit]` static constructor